test/jest/user-utils.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { didUserChange } from '../../src/user-utils';
+
+describe('user-utils', () => {
+    describe('#didUserChange', () => {
+        it('returns true if previousUser is null', () => {
+            expect(didUserChange(null, null)).toBeTruthy();
+        });
+
+        it('returns true if previousUser and newUser have different mpids', () => {
```

Comment:
please add another test to return false if the user's new mpid and old mpid are the same